### PR TITLE
Add e-mail forward boolean field and SQL view

### LINKED ISSUE #264: Add CH mail forward configuration
Add a `BooleanField` (a checkbox) to enable forwarding from a CH mail address to the person's Dienst2 email address. This way, users don't have to configure forwarding themselves and we don't have to maintain their mailboxes.

To be able to configure this on the mail server, we should create a SQL VIEW that can be queried by Exim with appropriate permissions.

### PR DESCRIPTION
**Pull request by praseodym**
Closes #264

### CODE REVIEW

File: ldb/models.py [hunk 1]
```diff
@@ -331,6 +335,19 @@ def age(self):
         else:
             return today.year - born.year
 
+    def clean(self):
+        if self.email_forward:
+            if self.ldap_username is None:
+                raise ValidationError(
```
Comment (jgadelange) [inline ldb/models.py:341]: Repace with `self.add_error(None, "CH LDAP username should be set to enable forwarding.")` or `self.add_error("email_forward", "...")`

https://docs.djangoproject.com/en/3.0/ref/forms/api/#django.forms.Form.add_error

File: ldb/templates/ldb/person_detail.html [hunk 2]
```diff
@@ -103,17 +103,29 @@ <h2 class="page-header">
...
+                <label class="col-sm-4 control-label">LDAP-gebruikersnaam</label>
+                <div class="col-sm-8">
+                  <p class="form-control-static">
+                    {% if user|has_group:'userman2' %}
+                      <a href="https://userman2.ch.tudelft.nl/users/{{ person.ldap_username }}">{{ person.ldap_username }}</a>
+                    {% else %}
+                      {{ person.ldap_username }}
+                    {% endif %}
+                    {% if person.email_forward %}
+                      <br/><small>E-mail wordt doorgestuurd naar {{ person.email }}</small>
```
Comment (jgadelange) [inline ldb/templates/ldb/person_detail.html:117]: Hmmmm... is this the most logical place to show the mail forward?

I think this should be in a `{% trans %}` template tag, so we are prepared for when we should switch to English

File: ldb/models.py (hunk 1 above)
Comment (praseodym) [reply ldb/models.py:341]: Hmm: AttributeError, 'Person' object has no attribute 'add_error'

Comment (praseodym) [reply ldb/models.py:341]: We can do a `raise ValidationError` for a field though: https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.clean

File: ldb/templates/ldb/person_detail.html (hunk 2 above)
Comment (praseodym) [reply ldb/templates/ldb/person_detail.html]: Updated :slightly_smiling_face:

File: ldb/models.py (hunk 1 above)
Comment (jgadelange) [reply ldb/models.py:341]: Ah, that's for a form. 🙈

**Pull request merged**